submissions/UA-girl/Task-HTML-CSS/index.html

```diff
@@ -0,0 +1,1538 @@
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Львів</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Odesa"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Одеса</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Dnipro"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Дніпро</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Kharkiv"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Харків</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Kharkiv"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Полтава</span>
+              </label>
+            </fieldset>
+            <div class="booking-details">
+              <span class="booking-ico booking-ico__place">1</span>
+              <span>Вільне місце</span>
+              <span
+                class="booking-ico booking-ico__place booking-ico__place__disabled"
+                >1</span
+              >
+              <span>Недоступне</span>
+            </div>
+            <fieldset class="booking-row">
+              <legend class="form-header">Місце:</legend>
+              <fieldset class="booking-row__inputs">
+                <legend class="booking-row__header">Ряд 1</legend>
+                <label class="booking-label booking-label__place">
+                  <input
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="1 1"
+                  />
+                  <span class="booking-ico booking-ico__place">1</span>
+                  <span class="visually-hidden">1 ряд 1 місце</span>
```

Comment:
If you trying to implement a11y. In this case, screen reader will read "1 1ряд 1 місце". Mabey code from line 242 better to hide from him to avoid duplicating reading row numbers?

Reply:
Yes, I've edded <span class="visually-hidden">1 ряд 1 місце</span> for screenreaders. Fixed.